src/ResourceManager/DataLakeStore/Commands.DataLakeStore/help/Get-AzureRmDataLakeStoreChildItemProperties.md

```diff
@@ -0,0 +1,260 @@
+---
+external help file: Microsoft.Azure.Commands.DataLakeStore.dll-Help.xml
+Module Name: AzureRM.DataLakeStore
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmDataLakeStoreChildItemProperties
+
+## SYNOPSIS
+Gets the properties (Disk usage and Acl) for the entire tree from the specified path
+
+## SYNTAX
+
+### GetDiskUsage
+```
+Get-AzureRmDataLakeStoreChildItemProperties [-Account] <String> [-Path] <DataLakeStorePathInstance>
+ [-OutputPath] <String> [-SaveToAdl] [-IncludeFiles] [-MaximumDepth <Int32>] [-Concurrency <Int32>]
+ [-GetDiskUsage] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### GetAllProperties
+```
+Get-AzureRmDataLakeStoreChildItemProperties [-Account] <String> [-Path] <DataLakeStorePathInstance>
+ [-OutputPath] <String> [-SaveToAdl] [-IncludeFiles] [-MaximumDepth <Int32>] [-Concurrency <Int32>]
+ [-GetDiskUsage] [-GetAcl] [-HideConsistentAcls] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### GetAclDump
+```
+Get-AzureRmDataLakeStoreChildItemProperties [-Account] <String> [-Path] <DataLakeStorePathInstance>
+ [-OutputPath] <String> [-SaveToAdl] [-IncludeFiles] [-MaximumDepth <Int32>] [-Concurrency <Int32>] [-GetAcl]
+ [-HideConsistentAcls] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
```

Comment:
@rahuldutta90 please fill out a description for this cmdlet

Reply:
@cormacpayne Fixed